evap/evaluation/urls.py

```diff
@@ -8,6 +8,7 @@
 urlpatterns = [
     url(r"^$", index, name="index"),
     url(r"^faq$", faq, name="faq"),
+    url(r"^set_language", set_and_save_language, name="set_lang"),
```

Comment:
this was originally done to avoid name collisions with the i18n function set_language. Will fix